packages/events/defaultSubscribers/migrate.js

```diff
@@ -0,0 +1,24 @@
+const OS = require("os");
+
+module.exports = {
+  initialization: function () {
+    this.logger = console;
+  },
+  handlers: {
+    "migrate:start": [
```

Comment:
OK, something really does not make sense here.  You've added a handler for `"migrate:start"`... but you haven't added anything to actually *emit* this event, nor have you removed any existing code that would perform this printout manually.  What's going on here?

Reply:
remove migrate:start